### DISCUSSION TITLE
the construction of molecular models using the Martini 3 force field

### DISCUSSION THREAD

**Discussion by zhangtrzhangtr** [ricalessandri/Martini3-small-molecules#42] [Q&A] [answered]
I have some inquiries regarding the construction of molecular models using the Martini 3 force field. Could you kindly provide some assistance or guidance on this matter? I am interested in constructing a thiophene five-membered ring with one of the hydrogen atoms replaced by a fluorine atom. I have noted that the original thiophene ring can be mapped as ((TC5)2-TC6). I would like to replace one of the TC5 beads, but I am unable to find in the mapping table how to replace a ring carbon atom and a fluorine atom with a single bead. Specifically, for the structure −CHa)=C(−F)−CHa)=, what type of bead would be most appropriate for this replacement? I would greatly appreciate your assistance in this matter and thank you very much for helping me

**Comment by ricalessandri** [comment, marked as answer]
Hi @zhangtrzhangtr , this hasn't been really tested, that's why there's no explicit option for this case in [Table 1 of the Martini 3 small molecule paper](https://onlinelibrary.wiley.com/doi/pdf/10.1002/adts.202100391). I would suggest to substitute the TC5 bead that maps one of the −CHa)=C(-H)−CHa)= in thiophene with a TX4e to map the −CHa)=C(−F)−CHa)= group. This choice is based on the SX4e used in Benzotrifluoride ([Table 1 of the Martini 3 small molecule paper](https://onlinelibrary.wiley.com/doi/pdf/10.1002/adts.202100391)) and in general the use of X4e for organofluorides ([Supplementary Table 24 of the Martini 3 paper](https://www.nature.com/articles/s41592-021-01098-3#Sec13)). I hope this helps and feel free to follow up.

**Reply by zhangtrzhangtr** [reply]
Thank you for your guidance. I will initially try using the TC6-TC5-TX4e mapping for the structure in my calculations. I appreciate your help greatly.
